_issues/emails.md

```diff
@@ -1,23 +1,23 @@
 ---  
 layout: post  
-title: "But isn't she a criminal for using a private email server?"  
+title: "Hillary is a criminal for using a private email server?"  
 categories: controversy
 ---  
 
 Source: [What the FBI Files Reveal About Hillary Clinton’s Email Server](http://www.politico.com/magazine/story/2016/09/hillary-clinton-emails-2016-server-state-department-fbi-214307)
  
-Q: Isn't Hillary is a criminal because she used a private email server?
```

Comment:
I personally still like the Q & A for the first two paragraphs. But that's just me.